pkg/app/piped/platformprovider/kubernetes/applier.go

```diff
@@ -35,21 +35,36 @@ type Applier interface {
 	ReplaceManifest(ctx context.Context, manifest Manifest) error
 	// Delete deletes the given resource from Kubernetes cluster.
 	Delete(ctx context.Context, key ResourceKey) error
+	// Install does installing a release or upgrading it if it already exists.
+	Install(ctx context.Context) error
```

Comment:
To be honest, I feel a bit weird about the use of this interface. Most of the functions this interface contains are used for applying/creating resources based on loaded/manipulated manifests loader, this Install function does not really relate to applying tasks. Since we haven't know how we're using this interface anyway, I suggest separating this Intall interface instead of making it part of Applier. wdyt?

Reply:
@khanhtc1202 
I don't think it is too strange to have the install interface in the applier in terms of applying resources, but I understand the feeling.

Instead of that, how about creating the new interface, Installer?
```
type Installer interface {
	// Install the chart to Kubernetes.
	Install(ctx context.Context) error
}
```

Reply:
> To have the install interface in the applier in terms of applying resources.

I understand this feeling as well 😂 I think the hard part to make the decision for this is we don't know how it's used on the caller side, so how about to implement that part first. TBH, instead of adding not yet used interface, I think reviewing from how it would be used and define a NOT YET implemented interface is easier and we can confirm the follow works before actually implement the underlying execution. wdyt? 👀

Reply:
@khanhtc1202 
I've just realized that the way installing release with helm cannot add built-in annotations to manifest because not using manifests in it.
Hence we have to think about this again before implementing this.

Reply:
Oops, okay 🙆‍♂️